olive/resource_path.py

```diff
@@ -219,6 +219,14 @@ def save_to_dir(self, dir_path: Union[Path, str], name: str = None, overwrite: b
         # copy the resource to the new path
         if is_file:
             shutil.copy(self.config.path, new_path)
+
+            # Copy the data file, if exists
```

Comment:
For onnx model, if the external data's file name is changed after the `copy`, onnx model can not be loaded successfully.

Can we load model back and run onnx.save again?

Reply:
The file name is the same as it was saved. The logic here is building the data file name/path merely by appending ".data" to the corresponding onnx file path. This is still not a complete solution because the name of the data file is an input to the onnx.save method. None of the logic actually uses this but there is potential for this to fail.

This is a stop-gap solution to make the distributed onnx export work correctly. The correct solution need discussion as there are other scenarios that need to be covered as well (like weights for each operator is exported out as an independent file in which case the model is neither just one file (onnx), nor two files (onnx and onnx.data) but an entire folder with any number of files.